MusicBandSimulator/MusicBandSimulator/IMusicInstrument.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MusicBandSimulator
+{
+    /// <summary>
+    /// Інтерфейс, який містить Назву інструмента та звук, з яким грає цей інструмент.
+    /// </summary>
+    internal interface IMusicInstrument
+    {
+        public string Name { get; set; }
```

Comment:
remove seter, so nobody can change the name of instrument

Reply:
applies to all instruments

Reply:
public can also be removed, all interface members are public

Reply:
nice info. ty, didn't know that

Reply:
done